policy/modules/system/authlogin.if

```diff
@@ -137,6 +137,25 @@ interface(`auth_read_pam_motd_dynamic',`
 	allow $1 pam_motd_runtime_t:file read_file_perms;
 ')
 
+########################################
+## <summary>
+##	Read files in /run/motd.d directory
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`auth_read_pam_motd_files',`
```

Comment:
I recently added the interface 'auth_read_pam_motd_dynamic' which looks very similar to this - see #626 (which looks to be just above this interface).  Probably should combine.

Reply:
ok. I'll do a new pr for this.